extendeddatacrossword.go

```diff
@@ -415,10 +359,39 @@ func noMissingData(input [][]byte) bool {
 	return true
 }
 
+func numOnes(input [][]byte) int {
```

Comment:
Is this function used anywhere? If not, is there a reason to add it? Looks like we should delete it.

Reply:
Ah shoot I ported the bitmatrix functions over than forgot to remove that now-unused now 😂